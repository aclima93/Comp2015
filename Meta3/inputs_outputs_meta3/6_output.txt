Line 12, col 2: Symbol a2 not defined
