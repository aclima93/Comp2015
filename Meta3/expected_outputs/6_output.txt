===== Outer Symbol Table =====
boolean	_type_	constant	_boolean_
integer	_type_	constant	_integer_
real	_type_	constant	_real_
false	_boolean_	constant	_false_
true	_boolean_	constant	_true_
paramcount	_function_
program	_program_

===== Function Symbol Table =====
paramcount	_integer_	return

===== Program Symbol Table =====
a1	_integer_
b1	_integer_
c1	_integer_
myfunction	_function_

===== Function Symbol Table =====
myfunction	_integer_	return
a2	_integer_
b2	_integer_
c2	_integer_
